code/controllers/subsystem/timer.dm

```diff
@@ -681,11 +682,36 @@ SUBSYSTEM_DEF(timer)
 		return TRUE
 	return FALSE
 
+/**
+ * Fetches a timer
+ * Only really acceptable in debug situations, should NEVER use this
+ * I will hunt you down
+ *
+ * Arguments:
+ * * id a timerid or a /datum/timedevent
+ * * timer_subsystem - Optional, the subsystem to pull from. Replaced w SStimer if none is given
+ */
+/proc/gettimer(id, datum/controller/subsystem/timer/timer_subsystem)
+	if (!id)
```

Comment:
maybe add like #IFNDEF DEBUG to_chat(world, "someone being a dickhead with gettimer()(")

Reply:
s mostly just a copypasta from the other procs, at this point I'd stacktrace tbh